### ISSUE TITLE
Gnome software unable to do system update

### ISSUE THREAD

**Issue by Froggy232** [ublue-os/main#191]
Hi,
Since a few weeks, my gnome-software only do update for flatpak, not system image.
I need to type "rpm-ostree update" in a terminal everyday.
Does someone have an idea?
Thanks you

**Comment by bsherman** [comment]
Can you please provided output of `rpm-ostree status -v` ?

**Comment by Froggy232** [comment]
Hi,
The output is here : https://pastebin.com/dFg3UQgG
Thanks you

**Comment by bsherman** [comment]
First, thank you for the output... I noticed this in the output... `AutomaticUpdates: disabled`
so, that's why there are not automatic updates coming to your system.

But as I was reading and writing the above I realized I'd missed that you specifically asked about **Gnome Software** so, I'm sorry for mis-reading.

I could be wrong, but I don't believe Gnome Software can handle updating an rpm-ostree system with an OCI image... which is what we are in ublue.
